apps/src/templates/curriculumCatalog/no_sections_to_assign_dialog.module.scss

```diff
@@ -0,0 +1,34 @@
+.dialogueContainer {
```

Comment:
Nit: 'dialogContainer' instead of 'dialogueContainer' since 'dialog' is used elsewhere in this PR